api/features/dataclasses.py

```diff
@@ -0,0 +1,19 @@
+import typing
+from dataclasses import dataclass
+
+
+@dataclass
+class EnvironmentFeatureOverridesData:
+    """
+    Dataclass to hold the information about the number of overrides for a given feature in a given
+    environment.
+    """
+
+    num_segment_overrides: int = 0
+    num_identity_overrides: typing.Optional[int] = None
+
+    def add_identity_override(self):
```

Comment:
Why do we not have `add_segment_override`

Reply:
Yep, good point, will do.